### ISSUE TITLE
Intel Wifi is hard blocked on boot

### ISSUE THREAD

**Issue by dballard** [QubesOS/qubes-issues#3235]
#### Qubes OS version:

R3.2 (started after upgrade to 4.9 kernel)

#### Affected TemplateVMs: 

net, dom0, fedora-24

---

### Steps to reproduce the behavior:

Turn on laptop

### Expected behavior:

Wifi works and is not listed as hard blocked under rfkill. Would expect that pressing the software wifi button (fn f11) would fix this

### Actual behavior:

Wifi is hard blocked. Wifi is listed as hard blocked. Can not undo

Except that with experimentation I discovered putting the laptop to sleep and waking it up fixes this. Suddenly the wifi is not hard blocked and is usable again

### General notes:

System 76 Gazelle laptop with Intel iwlwifi Wifi. Things were find under Linux 4.4 until I upgraded to 4.9

---

**Comment by andrewdavidwong** [comment]
Is this related to (or a duplicate of) #3217, #3049, or #3008?

**Comment by dballard** [comment]
Seems to be the opposite of #3049 :/

sys-net on boot.

```
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: Xen PCI mapped GSI18 to IRQ26
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: pci frontend enable msi failed for dev 0:0
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: Xen PCI frontend error: -22!
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: pci_enable_msi failed - -22
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: loaded firmware version 18.168.6.1 op_mode iwldvm
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: CONFIG_IWLWIFI_DEBUG enabled
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: CONFIG_IWLWIFI_DEBUGFS enabled
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: CONFIG_IWLWIFI_DEVICE_TRACING disabled
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: Detected Intel(R) Centrino(R) Wireless-N 1030 BGN, REV=0xB0
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0: RF_KILL bit toggled to disable radio.
Oct 27 07:56:09 sys-net kernel: iwlwifi 0000:00:00.0 wlp0s0: renamed from wlan0
Oct 27 07:57:03 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:57:03 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:57:03 sys-net kernel: iwlwifi 0000:00:00.0: Radio type=0x2-0x2-0x1
Oct 27 07:57:03 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:57:03 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:57:03 sys-net kernel: iwlwifi 0000:00:00.0: Radio type=0x2-0x2-0x1
Oct 27 07:59:06 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:59:06 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:59:06 sys-net kernel: iwlwifi 0000:00:00.0: Radio type=0x2-0x2-0x1
Oct 27 07:59:07 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:59:07 sys-net kernel: iwlwifi 0000:00:00.0: L1 Enabled - LTR Disabled
Oct 27 07:59:07 sys-net kernel: iwlwifi 0000:00:00.0: Radio type=0x2-0x2-0x1
```

For some reason the kernel on boot is toggling it to disabled

**Comment by andrewdavidwong** [comment]
This issue is being closed because:

 - This issue has the "Release 3.2 updates" milestone.
 - [Qubes OS 3.2 recently reached end-of-life (EOL).](https://www.qubes-os.org/news/2019/03/28/qubes-3-2-has-reached-eol/)
 - This issue has not been updated in over a year.

If anyone believes that this issue should be reopened, please let us know in a comment here.
